CMakeLists.txt

```diff
@@ -146,8 +146,12 @@ target_include_directories(
 
 if (MLX_BUILD_PYTHON_BINDINGS)
   message(STATUS "Building Python bindings.")
-  find_package(Python COMPONENTS Interpreter Development)
-  find_package(pybind11 CONFIG REQUIRED)
+  find_package(Python 3.8 COMPONENTS Interpreter Development.Module REQUIRED)
```

Comment:
Maybe the python version should be a range here ?

Reply:
Good point, I didn't realize this but I checked and it is actually a range (>= 3.8).